content/en/docs/instrumentation/erlang/propagation.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Propagation
+weight: 35
+---
+
+## Cross Service Propagators
```

Comment:
These are the defaults, so no, as long as you have the instrumentation package for the client you are using it will propagate (using tracecontext and baggage) even without any configuration.

Reply:
Hm, rereading it I don't like the wording at all. It doesn't mention defaults or that the configuration does the globally registered propagators.

Fixing it up.

Reply:
I've updated this page.